makefile

```diff
@@ -34,3 +34,9 @@ black.check:
 	@ARGS="--check --color --diff" make black
 
 ci: | black.check flake
+
+docs:
```

Comment:
Can you please check to see if there is a recommended way to adding a line of documentation to a makefile for a task that might not be obvious?

This might be useful for explaining what each task is supposed to do if the intention isn't immediately obvious.

Reply:
At the top of the document you can see that # comments are allowed. I have seen examples where people simply use comments and add echo's to it to show this in the terminal too(https://makefiletutorial.com/). 


In any case, I have added some comments to commands I am almost certain of what they do. Can you expand on the rest?